app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.controller.js

```diff
@@ -39,6 +39,7 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.create.controller',
     };
 
     $scope.isNew = isNew;
+    $scope.isAdd = isAdd;
```

Comment:
I wonder if we should change `isAdd` to `isPipeline` or `forPipelineConfig` - `isAdd` doesn't really tell me much about its purpose, or why I'd set it one way or the other.